spec/factories/case_contacts.rb

```diff
@@ -1,16 +1,18 @@
 FactoryBot.define do
+  # note: uses enum status column, see:
+  # https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#enum-traits
```

Comment:
I found this a bit hard to follow initially. It made sense that if there is an enum then traits are automagically created for it but I did not realize that `active`, `started` within the factories were calls to tho traits. 

I would appreciate like 2 sentences explaining that because I don't think is a pattern used elsewhere in the project.

This was the piece of context I was missing https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#traits-within-traits.

Reply:
That makes sense, will explain better!

Reply:
updated.